cves/kernel/CVE-2015-7566.yml

```diff
@@ -320,8 +322,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: True
+  note: Code was reviewed before it was committed.
```

Comment:
Are you able to provide a bit more detail into who contributed towards this fix?